.github/workflows/mkdocs.yml

```diff
@@ -0,0 +1,18 @@
+name: ci
+on:
+  push:
+    branches:
+      - master
+      - main
```

Comment:
Can we test this by including this/the `689-documentation` branch here for the moment?